tensorflow/lite/kernels/transpose_conv.cc

```diff
@@ -558,9 +572,29 @@ void EvalQuantizedPerChannel16x8(
                             weights->params.zero_point ||
                             output->params.zero_point;
 
-  // Fallback to reference kernel when bias_type is int64 as
```

Comment:
Why removing these comments?

Reply:
I think these comments are no longer relevant with these changes. I am happy to add these back in though.

Reply:
I think the fact that we doesn't have optimized kernel for int64 bias doesn't change after this cl.

Please add these comment at line #598.